integrations/dense/test_encode.py

```diff
@@ -58,6 +58,57 @@ def test_dpr_encode_as_faiss(self):
         self.assertAlmostEqual(hit[0].vectors[0], -6.88267112e-01, places=4)
         self.assertEqual(searcher.num_docs, 3204)
 
+    def test_dpr_encode_as_faiss_search_with_partitions(self):
+        # Create two partitions of the CACM index, search them individually, and merge results to compute top hit
+        index_dir = f'{self.pyserini_root}/temp_index'
+        os.makedirs(os.path.join(index_dir, 'partition1'), exist_ok=True)
+        os.makedirs(os.path.join(index_dir, 'partition2'), exist_ok=True)
+        self.temp_folders.append(index_dir)
+        cmd1 = f"python -m pyserini.encode input   --corpus {self.corpus_path} \
```

Comment:
single quote please...

note sure why the above f-string is in double quotes, maybe oversight from previous PR - can you fix that also?